controllers/C_InsuranceCompany.class.php

```diff
@@ -52,25 +53,23 @@ function list_action($sort = "")
     }
 
 
-    function edit_action_process()
+    public function edit_action_process()
     {
         if ($_POST['process'] != "true") {
             return;
         }
 
-        //print_r($_POST);
         if (is_numeric($_POST['id'])) {
             $this->icompanies[0] = new InsuranceCompany($_POST['id']);
         } else {
             $this->icompanies[0] = new InsuranceCompany();
         }
 
-        parent::populate_object($this->icompanies[0]);
+        $this::populate_object($this->icompanies[0]);
```

Comment:
this seems odd. was this to fix something?

Reply:
@stephenwaite If you are trying to get the current class's static method use the keyword **self** instead of $this.  self::populate_object($this->icompanies[0]);

Reply:
was emulating mentor Jerry :)

https://github.com/openemr/openemr/commit/3acf15c7377de3baf047d9f4a97f833dbc6f0a35#diff-9ae299bdc5a54b5b13f340513a40f8647a2c4d58eba1c84bd0523ba9b756307e

Reply:
@sjpadgett should [$this be self here?](https://github.com/openemr/openemr/blob/master/interface/forms/vitals/C_FormVitals.class.php#L151)